### ISSUE TITLE
Add multilingual support on create form

### ISSUE THREAD

**Issue by yvangodard** [tildaslash/RatticWeb#345]
Labels of the form used to add a password does not currently support the translation (see attached for example, in french : all labels are in english).
It would be interesting if this form is fully multilingual.
![capture d ecran 2015-03-16 a 08 51 41](https://cloud.githubusercontent.com/assets/469329/6662306/3f580ec0-cbba-11e4-8da1-b6c333d7dd59.jpg)

**Comment by smarthall** [comment]
Good catch, looks like most  of the forms do this, so lets do all the forms.

**Comment by smarthall** [comment]
For anyone who wants to take this on as their first bug what needs to happen is that we need to add a `verbose_name` argument to each field with a translation string. So to do this make sure this line is in the imports section for the `models.py` file:
```

**Comment by smarthall** [comment]
Then finally run `./manage.py makemessages` to update the translations file so we can give the strings to our translators.
